src/sentry/audit_log/services/log/impl.py

```diff
@@ -73,6 +81,28 @@ def find_last_log(
 
         return last_entry.as_event()
 
+    def _should_skip_invalid_event(self, event: AuditLogEvent) -> bool:
+        event_id_pass_list = self._get_invalid_event_id_pass_list()
+        return event.event_id in event_id_pass_list
+
+    def _get_invalid_event_id_pass_list(self) -> list[int]:
+        pass_list = options.get(self.event_id_skip_list_option)
+        list_valid = isinstance(pass_list, list)
+
+        if list_valid:
+            for item in pass_list:
+                if not isinstance(item, int):
+                    list_valid = False
+                    break
+
+        if not list_valid:
+            sentry_sdk.capture_message(
+                f"Invalid audit_log skip list. Verify that the '{self.event_id_skip_list_option}' option is a list of ints."
+            )
```

Comment:
If you use `logging.error` then you can pass `extra={"pass_list": pass_list}` here